src/main/java/uk/gov/hmcts/divorce/divorcecase/model/FinalOrder.java

```diff
@@ -173,6 +186,119 @@ public enum Granted implements HasLabel {
     )
     private TranslatedToLanguage applicant2FinalOrderLateExplanationTranslatedTo;
 
+    @CCD(
+        label = "Explain why you need to apply for the final order",
+        access = {Applicant2Access.class},
+        typeOverride = TextArea
+    )
+    private String applicant2SolFinalOrderWhyNeedToApply;
+
+    @CCD(
+        label = "Respondent Solicitor Applied For Final Order",
```

Comment:
As discussed yesterday, change label text as first word Init cap and rest lower caps to match other labels

Reply:
done